### PR TITLE
OKTA-626096 Release notes for 2023.07.0

### PR DESCRIPTION
**Pull request by vanngo-okta**
## Description:
- **What's changed?** Release notes for 2023.07.0
- **Is this PR related to a Monolith release?** No, 2023.07.0

### Resolves:

* [OKTA-626096](https://oktainc.atlassian.net/browse/OKTA-626096)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/release-notes/2023-okta-identity-engine/index.md [hunk 1]
```diff
@@ -4,6 +4,51 @@ title: Okta Identity Engine API Products release notes 2023
...
+
+#### Developer documentation update in 2023.07.0
+
+The Primer for OIN OpenID Connect guides have been removed. Content for OIDC protocol requirements, multi-tenancy, and best practices have been updated and merged to the [Overview of Single Sign-On in the OIN](/docs/guides/oin-sso-overview/) and [Build an SSO integration](/docs/guides/build-sso-integration/openidconnect/main/) guides. <!-- OKTA-447961 -->
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2023-okta-identity-engine/index.md:45]:
```suggestion
The Primer for OIN OpenID Connect guides has been removed. Content for OIDC protocol requirements, multi-tenancy, and best practices have been updated and merged to the [Overview of Single Sign-On in the OIN](/docs/guides/oin-sso-overview/) and [Build an SSO integration](/docs/guides/build-sso-integration/openidconnect/main/) guides. <!-- OKTA-447961 -->
```

File: packages/@okta/vuepress-site/docs/release-notes/2023/index.md [hunk 2]
```diff
@@ -2,6 +2,35 @@
...
+
+#### Developer documentation update in 2023.07.0
+
+The Primer for OIN OpenID Connect guides have been removed. Content for OIDC protocol requirements, multi-tenancy, and best practices have been updated and merged to the [Overview of Single Sign-On in the OIN](/docs/guides/oin-sso-overview/) and [Build an SSO integration](/docs/guides/build-sso-integration/openidconnect/main/) guides. <!-- OKTA-447961 -->
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2023/index.md:28]:
```suggestion
The Primer for OIN OpenID Connect guides has been removed. Content for OIDC protocol requirements, multi-tenancy, and best practices have been updated and merged to the [Overview of Single Sign-On in the OIN](/docs/guides/oin-sso-overview/) and [Build an SSO integration](/docs/guides/build-sso-integration/openidconnect/main/) guides. <!-- OKTA-447961 -->
```

**Review by susanharper-okta — APPROVED** [review]
couple edits and my faux pas - approving to not hold you up

File: packages/@okta/vuepress-site/docs/release-notes/2023-okta-identity-engine/index.md [hunk 3]
```diff
@@ -4,6 +4,51 @@ title: Okta Identity Engine API Products release notes 2023
 
 <ApiLifecycle access="ie" />
 
+## July
+
+### Monthly release 2023.07.0
+
+| Change | Expected in Preview Orgs |
+| ------ | ------------------------ |
+| [Front-channel Single Logout is EA in Preview](#front-channel-single-logout-is-ea-in-preview) | July 12, 2023 |
```
[the remark this answers is not in the document]
Comment (vanngo-okta) [reply packages/@okta/vuepress-site/docs/release-notes/2023-okta-identity-engine/index.md:13]: We should add it. 
I'll make the change. Thanks for letting me know!

File: packages/@okta/vuepress-site/docs/release-notes/2023-okta-identity-engine/index.md [hunk 4]
```diff
@@ -4,6 +4,51 @@ title: Okta Identity Engine API Products release notes 2023
...
+The Policy API Self-Service Password Reset object can now use Google Authenticator to initiate recovery scenarios. Previously, the object could only use the Email, Phone, or Okta Verify authenticators to initiate recovery. This addition enhances the user experience by increasing the number of options available for recovery.  See [Policy API](/docs/reference/api/policy/#self-service-password-reset-action-object). <!-- OKTA-609191 IDX_SSPR_EXTENDED_PRIMARY_FACTORS-->
+
+#### Okta-generated client secret length increase
+
+The length of Okta-generated client secrets has been increased from 40 to 64 characters.
+<!-- OKTA-619134 -->
+
+#### Smart Card authenticator is Self-Service GA in Preview
+
+You can add a new Smart Card authenticator that enables PIV to be used in authentication policies. You can also restrict the authentication policies to use only Smart Card authenticator as MFA. See `properties.additionalAmr` in the updated [Identity Provider](https://developer.okta.com/docs/reference/api/idps/#identity-provider-attributes) attributes for `X509` Smart Card support.
```
[the remark this answers is not in the document]
Comment (vanngo-okta) [reply packages/@okta/vuepress-site/docs/release-notes/2023-okta-identity-engine/index.md:36]: PIV == Personal Identity Verification
https://help.okta.com/oie/en-us/Content/Topics/Security/idp-smart-card-end-user.htm
I recall asking PM/Dev lead about this once, and I think they mentioned that it's synonymous with fed smart cards. Should we spell it out?

**Review by susanharper-okta — APPROVED** [review]
LGTM 🚀

**Pull request merged**